README.md

```diff
@@ -20,8 +20,14 @@ This project is licensed under the BSD-3-Clause License.
 3. Install [Jekyll](https://jekyllrb.com/)
 4. Install dependencies: `bundle install`
 5. Build: `bundle exec jekyll serve` for the local server, `bundle exec jekyll build` for one off builds. Either way, the HTML of the site is generated to `/_site`
-6. Point your browser at `http://127.0.0.1:4000/`
```

Comment:
These instructions seem to imply that I still need to install all the dependencies on my local environment, _then_ run docker. I think it's not a win :) Does it work if I run `docker-compose` as step 2? If so let's make the doc reflect that?

Reply:
Agreed.

Reply:
Hi @dblock I totally agree with you. I didn't want to remove the steps already preset there so that people may decide if they want to install it natively or via docker.  But I do agree that this is a bit misleading and we should make it clear on the How To on README. Will try to revise it and feel free to comment/suggest.

Reply:
I made some changes here https://github.com/opensearch-project/project-website/pull/34/commits/feed774ceb28cf62a563021ae5a622955ae0b48a, but feel free to suggest changes so that it is clear when and how to use `docker-compose`.